doc/fvwm/options.xml

```diff
@@ -370,7 +370,23 @@ internal debugging and should only be used by developers.</para>
   <varlistentry>
   <term><option>-v</option></term>
   <listitem>
-<para>Enables debug logging.  Write to ~/fvwm3-output.log in append mode.</para>
+<para>Enables debug logging.  Writes in append mode to fvwm log file, 
+which is ~/.fvwm/fvwm3-output.log by default. See ENVIRONMENT section on 
+how to override this location using <envar>$FVWM_USERDIR</envar> or 
+<envar>$FVWM3_LOGFILE</envar>.</para>
+
```

Comment:
Again, `FVWM_USERDIR` is set once, and really shouldn't be updated once `Fvwm` is running.